dynamiqs/time_array.py

```diff
@@ -46,24 +46,24 @@ def pwc(times: ArrayLike, values: ArrayLike, array: ArrayLike) -> PWCTimeArray:
     window function defined by $\Omega_{[t_a, t_b[}(t) = 1$ if $t \in [t_a, t_b[$ and
     $\Omega_{[t_a, t_b[}(t) = 0$ otherwise, and $O_0$ is a constant array.
 
-    Notes:
-        The argument `times` argument must be sorted in ascending order, but does not
-        need to be evenly spaced.
-
-    Notes:
-        If the returned time-array is called for a time $t$ which does not belong to any
-        time intervals, the returned array is null.
-
     Args:
         times _(array_like of shape (N+1,))_: Time points $t_k$ defining the boundaries
             of the time intervals, where _N_ is the number of time intervals.
         values _(array_like of shape (..., N))_: Constant values $c_k$ for each time
             interval.
         array _(array_like of shape (n, n))_: Constant array $O_0$.
 
+    Notes-: `times` sorting
+        The argument `times` argument must be sorted in ascending order, but does not
+        need to be evenly spaced.
+
```

Comment:
Nothing should be in-between Args and Returns. Typically we put all notes on top of args, and I think it's better not to mix the function description and the args/returns section.